### DISCUSSION TITLE
Button

### DISCUSSION THREAD

**Discussion by meenow708** [iTwin/community#21] [General]
Hi,
 i wrote below code to add button which can redirect me to the my dashboard but the button is not appear on the view. 

can any one have a solution please?

import React from 'react';
import { Button, Flex } from '@itwin/itwinui-react';
import { UiItemsProvider } from "@itwin/appui-abstract";

export class DashboardButton implements UiItemsProvider {
  public readonly id = "DashboardButton";

  render() {
    const handleButtonClick = () => {
      // Redirect to the specified URL
      window.location.href = 'https://lookerstudio.google.com/embed/reporting/6466750c-4ed8-4892-9feb-19a9ccd60481/page/Qk6oD';
    };

    return (
      <Flex>
        <Button onClick={handleButtonClick}>Dashboard</Button>
      </Flex>
    );
  }
}

The below code is the app.tsx file code, i have attached the images the code which i have added to the app.tsx file,  please let me know.

![Screenshot 2024-02-01 165128](https://github.com/iTwin/community/assets/157709671/8a6a55ce-9bea-4feb-8448-934cf1edce13)

![image](https://github.com/iTwin/community/assets/157709671/703e73e2-1eef-4ef9-
![Screenshot 2024-01-31 153654](https://github.com/iTwin/community/assets/157709671/c3fd0399-54f0-4336-8428-1e94aee0232f)
b025-d5f304b37bb2)

It will be pleasure if anybody take a step to help!

**Comment by aruniverse** [comment]
It doesnt look like you correctly implemented the UiItemsProvider. Please have a look at the [AppUI Samples](https://developer.bentley.com/samples/)

**Comment by meenow708** [comment]
if u have any single button example for redirecting to any web or any dashboard than please show me!

![image](https://github.com/iTwin/community/assets/157709671/a1ead3ab-56e1-4133-b946-009164d20e46)
i want button that should reflect on this view.
